streaming/src/test/scala/org/apache/spark/streaming/scheduler/ReceiverSchedulingPolicySuite.scala

```diff
@@ -127,4 +127,26 @@ class ReceiverSchedulingPolicySuite extends SparkFunSuite {
       assert(executors.isEmpty)
     }
   }
+
+  test("when scheduleReceivers return a balanced result, we should not restart receivers") {
```

Comment:
Removed this test since we don't call rescheduleReceiver when the receiver registers at the first time.